src/api/app/models/kiwi/image/from_xml.rb

```diff
@@ -0,0 +1,91 @@
+module Kiwi
+  class Image
+    class FromXml
```

Comment:
I dislike that naming ... Verb + Noun... bad combination for a class. Maybe putting all together in one class?

Reply:
> I dislike that naming ... Verb + Noun... 

Can you explain why you don't like this?

> Maybe putting all together in one class?
 
No thats the whole point of this PR: to isolate independent code. I don't want the Kiwi stuff to end up looking like other parts of OBS where we just throw everything into one class without thinking about how to organise components of code.

Reply:
> Can you explain why you don't like this?

For me sounds more "natural" that class names are nouns and method names are verbs...

> No thats the whole point of this PR: to encapsulate independent pieces of code. I don't want the Kiwi stuff to end up looking like other parts of OBS where we just throw everything into one class without thinking about how to organise components of code.

I meant all the Xml handling for `Kiwi::Image`, maybe all together in the class `Kiwi::Image::Xml` and the ToXml part as the method `build`and the FromXml part as the method `parse`

Reply:
> I meant all the Xml handling for Kiwi::Image, maybe all together in the class Kiwi::Image::Xml and the ToXml part as the method exportand the FromXml part as the method parse

I know, I think that the ToXml and FromXml parts are independent and complex enough that they each deserve their own class.

Reply:
I have to insist though that To and From aren't verbs but prepositions - 'a word governing, and usually preceding, a noun or pronoun and expressing a relation to another word'.

your regular grammar >random N word<

Reply:
Thanks @coolo 😈 

BTW, do you have any opinion on the naming?

Reply:
ok, what about `Kiwi::Image::XmlBuilder#build` for the `Kiwi::Image::ToXml#to_xml` and `Kiwi::Image::XmlParser#parser` for the `Kiwi::Image::FromXml#from_xml` ??

Reply:
> ok, what about Kiwi::Image::XmlBuilder#build for the Kiwi::Image::ToXml#to_xml and Kiwi::Image::XmlParser#parser for the Kiwi::Image::FromXml#from_xml ??

:+1: